webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -1213,9 +1217,13 @@ protected function processClaim($judging, Request $request, ?string &$claimWarni
                 $this->dj->auditlog($auditLogType, $auditLogId, $action . 'ed');
 
                 if ($action === 'claim') {
-                    return $this->redirectToRoute('jury_submission', ['submitId' => $judging->getSubmission()->getSubmitid()]);
+                    return $this->redirectToLocalReferrer($this->router, $request,
+                        $this->generateUrl('jury_submission', ['submitId' => $judging->getSubmission()->getSubmitid()])
+                    );
                 } else {
-                    return $this->redirectToRoute('jury_submissions');
+                    return $this->redirectToLocalReferrer($this->router, $request,
+                        $this->generateUrl('jury_submissions')
+                    );
```

Comment:
Maybe (?) symmetrically, if you unclaim something you're done with it and will return to the list

Reply:
Discussed with @eldering and @meisterT and we think that is not better. So I will leave this part as this PR does it.